app/helpers/shop_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ShopHelper
```

Comment:
@andela-tpeters make it a habit not to commit anything not useful to your feature